### PR TITLE
Sort ascending

### PR DESCRIPTION
**Pull request by Ansper11**
Sorting in Ascending order: To use this function, we need to create a Int_ascending object and use object.ascending(input value). The input value is an unsorted integer array. The function will sort the input array in ascending order. and Test_int_ascending class use a JUnit test to ensure the function can work properly

### CODE REVIEW

File: src/Int_ascending.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+public class Int_ascending {
+
+	
+	public static void ascending(int[]arr) {
```
Comment (eloiseespel) [inline src/Int_ascending.java:5]: This is great code Andy! Just so you have a comment/change to make for the assignment you could add a header to your ascending method to explain what it does and what arguments it takes

**Review by Ansper11 — COMMENTED** [review]
add some comments

**Review by yyht6300a — APPROVED** [review]
The code is perfect! If you want, you can add some comments to explain how the sorter working.

**Review by Nick-McGee — COMMENTED** [review]
Easy to understand test. Maybe just briefly comment the algorithm used, or if it's a unique one explain it.

**Comment by yyht6300a** [comment]
Nice code!

**Comment by Ansper11** [comment]
Thanks

**Pull request merged**